server/src/test/java/org/apache/iotdb/db/integration/IoTDBAutoCreateSchemaIT.java

```diff
@@ -24,10 +24,7 @@
 import org.apache.iotdb.jdbc.Config;
 import org.apache.iotdb.jdbc.IoTDBSQLException;
 
-import org.junit.After;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.*;
```

Comment:
generate , we should not use import *

Reply:
Actually, all the unit under this package has been used and IDEA auto simplifies the import.